src/_balancer/utils.ts

```diff
@@ -41,18 +40,13 @@ export async function subgraphRequest(url, query) {
 }
 
 export function getTokenLogoUrl(address: string): string | null {
-  if (logos.includes(address.toLowerCase()))
-    return `https://raw.githubusercontent.com/balancer-labs/assets/master/assets/${address.toLowerCase()}.png`;
-  let trustwalletId: string | null = null;
   if (address === 'ether') {
-    trustwalletId = '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2';
-  } else {
-    const checksum = getAddress(address);
-    const token = config.tokens[checksum];
-    if (token && token.hasIcon) {
-      trustwalletId = checksum;
-    }
+    address = config.addresses.weth;
   }
-  if (!trustwalletId) return null;
-  return `https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/${trustwalletId}/logo.png`;
+  address = getAddress(address);
+  const metadata = config.tokens[address];
+  if (!metadata) {
+    return `https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/${address}/logo.png`;
```

Comment:
This assume there will always be an image available but it's not always the case. If you add custom token for example `0x1e1100070f3c0414e7d5e797ca062e2244178b8f` the image will be broken. I'd rather return null here so it will show the jazzicon.
```suggestion
    return null;
```

Reply:
If we return null, that would avoid loading icons for custom tokens. What if we handle that in `onerror`?

Reply:
@Destiner Yes let's go for `onerror` we will cover more logos this way.